### DISCUSSION TITLE
How to Remove Members from Sidebar

### DISCUSSION THREAD

**Discussion by brightlee6** [mediacms-io/mediacms#463] [Q&A]
Any idea on how to remove Members from Sidebar?

Should I change and update static/js/_commons.js? But this file is very difficult to understand and almost impossible to edit, as it has only one line with all contents.

Thanks.

**Comment by brightlee6** [comment]
Any helps or suggestions? Thanks.

**Reply by mgogoulos** [reply]
edit `templates/config/installation/site.html` and remove this section

```
-           members:{
-               title: 'Members',
-           },
```

then restart mediacms and Members won't be displaying

**Reply by petr0ff** [reply]
@mgogoulos hi! Another related question: how to remove 'About' and 'Terms' from sidebar? I can't find them in any of django templates..
